libs/testing/src/integration.service.ts

```diff
@@ -80,6 +80,28 @@ export class IntegrationService {
   }
 
   async createChannelIntegrations(environmentId: string, organizationId: string) {
+    const novuMailPayload = {
+      _environmentId: environmentId,
+      _organizationId: organizationId,
+      providerId: EmailProviderIdEnum.Novu,
+      channel: ChannelTypeEnum.EMAIL,
+      credentials: {},
+      active: false,
+    };
+
+    await this.integrationRepository.create(novuMailPayload);
+
+    const novuSmsPayload = {
+      _environmentId: environmentId,
+      _organizationId: organizationId,
+      providerId: SmsProviderIdEnum.Novu,
+      channel: ChannelTypeEnum.SMS,
+      credentials: {},
+      active: false,
+    };
+
+    await this.integrationRepository.create(novuSmsPayload);
```

Comment:
They can be executed in parallel (we are not doing any kind of error management so it would work the same if any failure).

Reply:
This is in test that why I did not do any error management.